### DISCUSSION TITLE
About developing the dbeaver plug-in

### DISCUSSION THREAD

**Discussion by yangmeng66** [dbeaver/dbeaver#16900] [Ideas]
1 We want to develop plug-ins for our company's self-developed database based on dbeavver. If it is successful, we want to mention the master of dbeaver. Is this OK?

2. Is there any documentation about the debugging development of dbeaver

**Comment by justinSmileDate** [comment]
Hello, have you found the relevant documents?

**Comment by LonwoLonwo** [comment]
Hello. Take a look at the following:
https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse
https://github.com/dbeaver/dbeaver/wiki/Develop-in-IDEA
https://github.com/dbeaver/dbeaver/pull/13449
https://github.com/dbeaver/dbeaver/pull/18299
